apps/admin-gui/src/app/shared/components/dialogs/add-role-dialog/add-role-dialog.component.html

```diff
@@ -1,47 +1,66 @@
 <div class="{{theme}}">
-  <h1 mat-dialog-title>{{'DIALOGS.ADD_ROLE.TITLE' | translate}}</h1>
+  <div mat-dialog-title>
+    <h1 mat-dialog-title>{{'DIALOGS.ADD_ROLE.TITLE' | translate}}</h1>
+    <perun-web-apps-immediate-filter
+      *ngIf="selectedRule?.primaryObject"
+      (filter)="applyFilter($event)"
```

Comment:
I know that we use it in many places in GUI like this, but you can simplify it (so you don't need a special method in the ts file anymore).
```suggestion
      (filter)="filterValue = $event"
```